src/qt/sendcoinsdialog.cpp

```diff
@@ -313,14 +322,14 @@ bool SendCoinsDialog::PrepareSendText(QString& question_string, QString& informa
         formatted.append(recipientElement);
     }
 
-    if (model->wallet().privateKeysDisabled()) {
+    if (model->wallet().privateKeysDisabled() && !model->wallet().hasExternalSigner()) {
```

Comment:
As this condition is pretty logical and common now, maybe it should be given its own name e.g.  noPrivateKeys()

Reply:
`noPrivateKeys()` is a big ambiguous. It would have to be something like `noPrivateKeysAndNoExternalSigner()` which is pretty ugly...

Reply:
What about `noPrivateKeyAccess()` or something - the condition is that there are keys either in the wallet itself or that the wallet can 'use' via the external signer. Not a big deal though.